### DISCUSSION TITLE
Step-by-Step Tutorial Micro-Video Series

### DISCUSSION THREAD

**Discussion by registor** [lvjr/tabularray#405] [Show and tell]
Hello everyone,

I created a step-by-step tutorial video series recently that focused on the new interface of the tabularray.

Currently, the main content has been completed and 3/4 more videos are left in the future.

Your criticism and correction are welcome, thanks!

These videos have been uploaded at bilibili.

1. [introduction](https://www.bilibili.com/video/BV1ag4y1L7C8/)
2. [general interface](https://www.bilibili.com/video/BV1M14y1Z7LK)
3. [line(s) inner specifications](https://www.bilibili.com/video/BV1LM411V7HU)
4. [cell(s)/row(s)/column(s) inner specifications](https://www.bilibili.com/video/BV1R14y1Z7Fu)
5. [colspec/rowspec inner specifications](https://www.bilibili.com/video/BV1ro4y1x7Tz)
6. [misc inner specifications](https://www.bilibili.com/video/BV1Ps4y197pa)
7. [outers specifications](https://www.bilibili.com/video/BV14h4y1J7Rv)
8. [default of inners and outters](https://www.bilibili.com/video/BV1xh4y1J7Qk)
9. [orther tblr interfaces](https://www.bilibili.com/video/BV1vL41187gm)
10. [bases of longtblr environment](https://www.bilibili.com/video/BV1oo4y1x72E)
11. [outers of longtblr environment](https://www.bilibili.com/video/BV11o4y1x7xh)
12. [talltblr environment](https://www.bilibili.com/video/BV1jL411h7uD/)

These videos was created with Chinese, if you like the English srt is welcome.

**Comment by muzimuzhi** [comment]
```
* [introduction](https://b23.tv/zn557D9)
* [general interface](https://b23.tv/T5OJaSl)
* [line(s) inner specifications](https://b23.tv/mCQ2sdn)
* [cell(s)/row(s)/column(s) inner specifications](https://b23.tv/LmttmlP)
* [colspec/rowspec inner specifications](https://b23.tv/jKXp5Rq)
* [misc inner specifications](https://b23.tv/LfI4iLj)
* [outters specifications](https://b23.tv/WmBf1yE)
* [default of inners and outters](https://b23.tv/3U1Dxir)
* [orther based interfaces](https://b23.tv/d9bYtq0)
* [bases of longtblr environment](https://b23.tv/90AokpX)
* [outers of longtblr environment](https://b23.tv/6aMSC7x)
* [talltblr environment](https://b23.tv/49ANotx)
```

These short urls will expand to urls with all kinds of tracing parameters, for example
```
* [misc inner specifications](https://b23.tv/LfI4iLj)
==>
https://www.bilibili.com/video/BV1Ps4y197pa/?buvid=9F359AEB-7227-482A-A0B0-6792F7E5D27416817infoc&is_story_h5=false&mid=5AOlum3N3BBJqlfTDRflkQ%3D%3D&plat_id=147&share_from=ugc&share_medium=android&share_plat=android&share_session_id=b026e848-b26e-4b51-a905-bf1b48b3cd0f&share_source=COPY&share_tag=s_i&timestamp=1683667490&unique_k=LfI4iLj&up_id=355343550&vd_source=97ae3cf51ce90cad162edf93bc69f02b
```

I suggest to use clean form, like `https://www.bilibili.com/video/BV1Ps4y197pa`.

**Reply by registor** [reply]
Thanks, urls were updated.
